UnityProject/Assets/Scripts/Health/Sickness/PlayerSickness.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Assets.Scripts.Health.Sickness
```

Comment:
We don't really have file path name spaces, but it's good it's in a namespace

Reply:
What nomenclature of namespace do we use?

Reply:
There were no namespaces at all.
People started added namespaces recently when we changed to Mirror.
File path namespaces are good in my opinion.

Reply:
> There were no namespaces at all.
> People started added namespaces recently when we changed to Mirror.
> File path namespaces are good in my opinion.

I agree.  Also, with Visual Studio, it's added automagically when you create a new file :D

Reply:
How about ``Health.Sickness``?

Reply:
only complaint is maybe in the future were going to clean up the scripts folder so stuff might get moved around

Reply:
Yeah and using the system boundary as the namespace instead of the full path to script sounds better for me. I think Rider would suggest ``Health.Sickness`` instead of the full path.

Reply:
@ReiderJack @Bod9001 
Should I change the namespace like @corp-0  says?  Or do I keep it the way it is?

Reply:
Ok, will change it to Health.Sickness.  Both your argument makes a lot of sense.

Reply:
Changes in namespaces were made.  Tell me what you think.

Reply:
looks good